packages/config-dug/src/config-dug.ts

```diff
@@ -148,16 +150,25 @@ class ConfigDug<T extends ConfigDugSchema> extends EventEmitter {
     this.emit('config-reloaded', this.validatedValues);
   }
 
+  /**
+   * Force reloads the config from a specific plugin
+   *
+   * @returns Promise<void>
+   */
+  public async reloadPluginConfig(plugin: any): Promise<void> {
+    debug('reloading plugin config');
+
+    const pluginRawValues = await this.loadPluginConfig(plugin);
+
+    this.validateRawValues({ ...this.rawValues, ...pluginRawValues });
+
+    this.emit('plugin-config-reloaded', this.validatedValues);
+  }
+
   private async loadConfig(): Promise<void> {
     const environmentName = getEnvironmentName(this.options.envKey);
 
-    if (!this.pluginsInitialized) {
-      await this.initializePlugins();
-      this.pluginsInitialized = true;
-    }
```

Comment:
Moved this up to the load function to avoid having that `pluginsInitialized` variable that is doing something similar to what the `loaded` variable is.